### PR TITLE
docs: Added function examples

### PR DESCRIPTION
**Pull request by Franziska-Albrecht**
Added description and examples for the following functions: 
- Calculate length
- Centroid
- Compute extent
- Network expansion 
- Date extraction 

Smaller changes - wording - for: 
- Aggregate 
- interior point 
- Spatial join

### CODE REVIEW

File: cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml
Comment (Kate-Lyndegaard) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml:20]: ......that encloses.....        .......that enclose the source data geometries.....

File: cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml
Comment (Kate-Lyndegaard) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml:27]: ....source data geometries.....?

File: cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/network/help.xhtml
Comment (Kate-Lyndegaard) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/network/help.xhtml:25]: To define the buffer size using...

in the function, or in the function dialog?

File: cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/network/help.xhtml
Comment (Kate-Lyndegaard) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/network/help.xhtml:32]: To apply a mathematical expression, a variable must be added

File: cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml
Comment (Kate-Lyndegaard) [inline cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml:11]: ...enables users...   ....from a string type in the source data....

File: cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml
Comment (Kate-Lyndegaard) [inline cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml:14]: The date format must be specified by the user to acheive the correct result.

**Review by Kate-Lyndegaard — COMMENTED** [review]
As a side note, I am not sure if these changes will appear in the documentation once merged, as https://github.com/halestudio/hale/issues/873 is still open. I added documentation for aggregate, interiorPoint, spatialJoin and formattedString here: https://github.com/halestudio/hale/pull/854. Unfortunately, the changes are not visible, yet.

I've made a few suggestions, but Florian and Johanna may also have suggestions:-)

File: cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/centroid/help.xhtml
Comment (JohannaOtt) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/centroid/help.xhtml:15]: Protected Sites maybe in upper case because it that is how it is usually handled in INSPIRE?

File: cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml
Comment (JohannaOtt) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml:11]: Compute Extent is written with upper case, "calculate length" is written in lower case in its description. I think we should align along the documentation whether we want to spell our functions with upper or lower case.

File: cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/network/help.xhtml
Comment (JohannaOtt) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/network/help.xhtml:11]: upper/lower case?

File: cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/spatialJoin/help.xhtml
Comment (JohannaOtt) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/spatialJoin/help.xhtml:11]: Join - upper vs lower case?

Comment (JohannaOtt) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/spatialJoin/help.xhtml:11]: multiple datasets --> objects from multiple feature types?

File: cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml
Comment (JohannaOtt) [inline cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml:11]: upper vs lower case?

Comment (JohannaOtt) [inline cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml:11]: from a string type --> from a string? from an attribute using a string type?

Comment (JohannaOtt) [inline cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml:11]: target source --> target attribute?

File: cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml
Comment (JohannaOtt) [reply cst/plugins/eu.esdihumboldt.cst.functions.string/help/date/help.xhtml:14]: achieve?

**Review by JohannaOtt — CHANGES_REQUESTED** [review]
I think those are very helpful examples for the users!

File: cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml
Comment (JohannaOtt) [inline cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml:27]: "Convex Hull" --> "convex hull" if we decide to go with lower case for function/option names

**Review by JohannaOtt — CHANGES_REQUESTED** [review]
I only found one upper/lower case inconsistency, but I guess it would make sense to wait for Kate's review concerning English Feinheiten :)

**Review by Kate-Lyndegaard — APPROVED** [review]

**Review by JohannaOtt — APPROVED** [review]

**Pull request merged**
